### ISSUE TITLE
Allow users to select the default view for custom content tools

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#17877]
When a admin creates a custom content tool, we need to allow the admin to select the default view for the custom tool, whether it is shown as a list or as cards.
---
![Screen Shot 2020-01-21 at 1 54 45 PM](https://user-images.githubusercontent.com/934364/72834032-03fd3000-3c56-11ea-84db-6df93fad67ee.png)

**Comment by alfredo-dotcms** [comment]
- the endpoint POST http://localhost:8080/api/v1/portlet/custom, needs to save a new attribute called `dataViewMode` with values `list | card`

- a new endpoint needs to be created to get the saved value (dataViewMode)

**Comment by alfredo-dotcms** [comment]
CORE PR: https://github.com/dotCMS/core/pull/20595

**Comment by nollymar** [comment]
**Note to QA:**  Please test on several browser clients

**Comment by wezell** [comment]
We should get the default view in the jsp and not make a different REST call

```
	Map<String, String> initParams = (Map<String, String>) request.getAttribute("initParams");
	final String defaultView = initParams.getOrDefault("dataViewMode", "lastSelectedView");
	
```

And this view should override but NOT set the state.view variable

**Comment by nollymar** [comment]
Passed internal QA

**Comment by bryanboza** [comment]
Fixed, tested on release-21.07 // Postgres // FF... Now we are able to select the view for every tool group without problems
